hoptimator-operator/src/main/java/com/linkedin/hoptimator/operator/HoptimatorOperatorApp.java

```diff
@@ -92,7 +93,7 @@ public void run() throws Exception {
 
     Properties connectionProperties = new Properties();
     connectionProperties.putAll(properties);
-    K8sContext context = K8sContext.create(connectionProperties);
+    K8sContext context = K8sContext.create(new HoptimatorConnection(null, connectionProperties));
```

Comment:
Don't love this but now is required as the "connection" isn't really needed within the Operator